tools/pipelines/build-client.yml

```diff
@@ -98,23 +99,3 @@ extends:
     - ci:test:realsvc:tinylicious
     - ci:test:stress:tinylicious
     - test:copyresults
-    preCG:
-    - task: UseNode@1
-      displayName: Use Node 14.x
-      inputs:
-        version: 14.x
-    - task: Npm@1
-      displayName: npm ci
-      inputs:
-        command: 'custom'
-        customCommand: 'ci --ignore-scripts'
-        customRegistry: 'useNpmrc'
-    - task: Bash@3
-      displayName: 'Generate Mono repo package json'
-      inputs:
-        targetType: 'inline'
-        script: |
-          # Generate the package/package lock for the lerna project so we would scan it.
-          node node_modules/@fluidframework/build-tools/dist/genMonoRepoPackageJson/genMonoRepoPackageJson.js
```

Comment:
We need to move all release groups to pnpm before getting rid of the script and its folder, right?

Reply:
Yeah; for any release groups that still have lerna-lockfile.json we need to keep this task.